File: starter.client/src/app/models/product.ts
export class Product {
    constructor(name:string, description:string,price:number) {
        this.name = name;
        this.description = description;
        this.price = price;
    }
    id: number = 0;
    name!: string;
    description!: string;
    price!: number;
}

File: Starter.Server/Models/Entities/Product.cs
using System;
using System.Collections.Generic;

namespace Starter.Server.Models.Entities;

public partial class Product
{
    public int id { get; set; }
    public string? name { get; set; }
    public string? description { get; set; }
    public decimal? price { get; set; }
}
